docs/api/regressor/residuals.rst

```diff
@@ -3,58 +3,46 @@
 Residuals Plot
 ==============
 
-Residuals, in the context of regression models, are the difference between the observed value of the target variable (y) and the predicted value (ŷ), e.g. the error of the prediction. The residuals plot shows the difference between residuals on the vertical axis and the dependent variable on the horizontal axis, allowing you to detect regions within the target that may be susceptible to more or less error.
+Residuals, in the context of regression models, are the difference between the observed value of the target variable (y) and the predicted value (ŷ), i.e. the error of the prediction. The residuals plot shows the difference between residuals on the vertical axis and the dependent variable on the horizontal axis, allowing you to detect regions within the target that may be susceptible to more or less error.
 
-.. code:: python
+.. plot::
+    :context: close-figs
+    :alt: Residuals Plot on the Concrete dataset using a linear model
 
     from sklearn.model_selection import train_test_split
+    from sklearn.linear_model import Ridge
+    from yellowbrick.datasets import load_concrete
+    from yellowbrick.regressor import ResidualsPlot
 
-    # Load the data
-    df = load_data('concrete')
-
-    # Identify the feature and target columns
-    feature_names = [
-        'cement', 'slag', 'ash', 'water', 'splast', 'coarse', 'fine', 'age'
-    ]
-    target_name = 'strength'
-
-    # Separate the instance data from the target data
-    X = df[feature_names]
-    y = df[target_name]
+    # Load a regression dataset
+    X, y = load_concrete()
 
     # Create the train and test data
-    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)
-
-.. code:: python
-
-    from sklearn.linear_model import Ridge
-    from yellowbrick.regressor import ResidualsPlot
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)
 
     # Instantiate the linear model and visualizer
-    ridge = Ridge()
-    visualizer = ResidualsPlot(ridge)
+    model = Ridge()
+    visualizer = ResidualsPlot(model)
 
-    visualizer.fit(X_train, y_train)  # Fit the training data to the model
+    visualizer.fit(X_train, y_train)  # Fit the training data to the visualizer
```

Comment:
here you are fitting to the model.  `visualizer.fit(X_train, y_train)` actually calls sklearn estimator(Ridge) fit - @Kautumn06 correct me if I am wrong

Reply:
Interesting question! I suppose it does both, since [`fit` calls `score`](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/regressor/residuals.py#L463), which [then calls `draw`](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/regressor/residuals.py#L498)! In any case, I think it's okay to leave this as is